packages/strapi-helper-plugin/lib/internals/generators/container/class.js.hbs

```diff
@@ -73,7 +73,7 @@ const withConnect = connect(mapStateToProps, mapDispatchToProps);
 /* Remove this line if the container doesn't have a route and
 *  check the documentation to see how to create the container's store
 */
-const withReducer = injectReducer({ key: '{{ camelCase name }}', reducer, pluginId });
+const withReducer = strapi.injectReducer({ key: '{{ camelCase name }}', reducer, pluginId });
```

Comment:
Because it's the only way I found to have a single store for all the containers.

Before the injectReducer Hoc was in the `strapi-helper-plugin` so it was scoped by plugin instead of being global...

The only way I found is to put this hoc in the global strapi